src/index.html

```diff
@@ -7,8 +7,78 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+
   </head>
+  <header>
+    <nav class="header">
+      <div>
+        <img 
+        class="logo" 
+        src="./images/logo.png" 
+        alt="logo"
+        >
+      </div>
+      <ul class="header_list">
+
+        <li class="is-active">
+          <a class="text_bar is-active" href="index.html">
+            Apple
+            <div class="underline">
+            </div>
+          </a>
+        </li>
+        <li>
+          <a class="text_bar" href="index.html">
```

Comment:
```suggestion
          <a class="text_bar" href="#samsung">
```
you could mock href like this